statistics/sample_serial_test.go

```diff
@@ -0,0 +1,328 @@
+// Copyright 2017 PingCAP, Inc.
```

Comment:
Why change the test file name ?

Reply:
as the new name suggests, it make it clear that this test can only be run in serial.
this is some kind of `better practise` imported by others, i agree and follow.